stories/demos/components/Formatters/ChildRowDeleteButton.tsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import { RemoveCircle } from '@material-ui/icons';
```

Comment:
This would have removed the bundle size overhead in dev mode (tree shaking is ineffective in dev mode).

```jsx
import RemoveCircle from '@material-ui/icons/RemoveCircle';
```